ScheduleManagementApp/ScheduleManagementApp/UIKit+/UIViewController+.swift

```diff
@@ -19,4 +19,14 @@ extension UIViewController {
         alert.addAction(okButton)
         present(alert, animated: true)
     }
+    
+    // 화면 어느 곳을 터치하든 키보드 내리기
```

Comment:
```suggestion
    /// 화면 어느 곳을 터치하든 키보드 내리기
```
슬래시를 3개 작성해야 함수를 사용한 위치에서 함수의 summary를 볼 수 있습니다.